test/E2ETest/Microsoft.Test.E2E.AspNet.OData/DateTimeSupport/DateTimeController.cs

```diff
@@ -138,6 +138,36 @@ public ITestActionResult CopyFiles(int key, [FromBody]ODataActionParameters para
 
             return Ok(createdDate);
         }
+        [HttpGet]
+        public ITestActionResult GetModifiedDates(int key)
+        {
+            File file = _files.FirstOrDefault(f => f.FileId == key);
+            if (file == null)
+            {
+                return NotFound();
+            }
+            return Ok(file.ModifiedDates);
+        }
+
+        [HttpPost]
+        public ITestActionResult PostToModifiedDates(int key, [FromBody]DateTime newDateTime)
+        {
+            File file = _files.FirstOrDefault(f => f.FileId == key);
+            if (file == null)
+            {
+                return NotFound();
+            }
+            try
+            {
+                file.ModifiedDates.Add(newDateTime);
+
+            }
+            catch (Exception e)
+            {
+                string s = e.HelpLink;
+            }
+            return Updated(newDateTime);
```

Comment:
>newDateTime [](start = 27, length = 11)

it should be "file"?

Reply:
It should be file.ModifiedDates. Updated.